examples/Atlas/runAtlasManip.m

```diff
@@ -71,14 +71,12 @@ function runAtlasManip(example_options)
 manip_plan_data = QPLocomotionPlan.from_quasistatic_qtraj(r,PPTrajectory(qtraj_pp),...
   struct('bodies_to_track',[pelvis,r_hand],...
          'quat_task_to_world',repmat(uniformlyRandomQuat(),1,2),...
-         'translation_task_to_world',randn(3,2)));
+         'translation_task_to_world',randn(3,2),...
+         'track_com_traj_flag',true));
 r_arm_idx = r.findPositionIndices('r_arm');
 manip_plan_data.gain_set = 'manip';
 
-manip_plan_data.joint_pd_override_data = struct('position_ind',r_arm_idx,...
-                                              'kp',zeros(length(r_arm_idx),1),...
-                                              'kd',zeros(length(r_arm_idx),1),...
-                                              'weight',1e-5*ones(length(r_arm_idx),1));
+manip_plan_data.untracked_joint_ind = r_arm_idx;
```

Comment:
it might be clearer to change "ind" to "inds" (and in the LCM type) to make it clear that this is a list of indices, not a single index